google_shopping.js

```diff
@@ -1,5 +1,59 @@
 var data = require("./products.json")
+var items = data.items;
+var counter = 0;
+var images = [];
+var canon = [];
+var eBayCanon = [];
 
 // Write your solutions below
 
-console.log(data["items"]);
\ No newline at end of file
+// console.log(data["items"]);
+
+for( var i = 0; i < items.length; i++){
+	if(items[i].kind == "shopping#product"){
+		counter++;
+		console.log('found product');
+	}
+}
+console.log(counter);
+
+for(var i =0; i < items.length; i++){
+	if(items[i].product.inventories[0].availability == "backorder"){
+		console.log(items[i].product);
+	}
+}
+
+
+for( var i = 0; i < items.length; i++){
```

Comment:
This for loop won't achieve much, because the variables will be overwritten each time. I think you can take this part out.